src/web/app/components/copy-session-modal/copy-session-modal.component.spec.ts

```diff
@@ -30,4 +40,108 @@ describe('CopySessionModalComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with default fields', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+
+  const feedbackSessionToCopy: FeedbackSession = {
+    courseId: 'Test01',
+    timeZone: 'Asia/Singapore',
+    feedbackSessionName: 'Test session',
+    instructions: 'Answer all',
+    submissionStartTimestamp: 1555232400,
+    submissionEndTimestamp: 1555332400,
+    gracePeriod: 0,
+    sessionVisibleSetting: SessionVisibleSetting.AT_OPEN,
+    responseVisibleSetting: ResponseVisibleSetting.AT_VISIBLE,
+    submissionStatus: FeedbackSessionSubmissionStatus.OPEN,
+    publishStatus: FeedbackSessionPublishStatus.NOT_PUBLISHED,
+    isClosingEmailEnabled: true,
+    isPublishedEmailEnabled: true,
+    createdAtTimestamp: 1554967204,
+  };
+
+  const courseSessionIn: Course = {
+    courseId: 'Test01',
+    courseName: 'Sample Course 101',
+    creationDate: '13 Mar 2019',
+    deletionDate: 'Not Applicable',
+    timeZone: 'UTC',
+  };
+
+  const courseCopyTo: Course = {
+    courseId: 'Test02',
+    courseName: 'Sample Course 202',
+    creationDate: '14 Mar 2019',
+    deletionDate: 'Not Applicable',
+    timeZone: 'UTC',
+  };
+
+  it('should bind fields correctly', fakeAsync(() => {
```

Comment:
I think the test to verify whether an HTML element has the correct value or not can be done in one snapshot testing?